src/AvaTaxClient.php

```diff
@@ -6604,488 +7146,507 @@ class BatchFileModel
+     * @var string When the certificate will be/was expired
      */
-    public $sstPid;
+    public $expirationDate;
 
     /**
-     * @var string A unique code that references this company within your account.
+     * @var string File name for the certificate
      */
-    public $companyCode;
+    public $filename;
 
     /**
-     * @var string The name of this company, as shown to customers.
+     * @var boolean Is the certificate valid?
      */
-    public $name;
+    public $valid;
 
     /**
-     * @var boolean This flag is true if this company is the default company for this account. Only one company may be set as the default.
+     * @var boolean Is the certificate verified?
      */
-    public $isDefault;
+    public $verified;
 
     /**
-     * @var int If set, this is the unique ID number of the default location for this company.
+     * @var boolean The certificate is never renewed
      */
-    public $defaultLocationId;
+    public $neverRenew;
 
     /**
-     * @var boolean This flag indicates whether tax activity can occur for this company. Set this flag to true to permit the company to process transactions.
+     * @var boolean Is this certificate renewable?
      */
-    public $isActive;
+    public $renewable;
 
     /**
-     * @var string For United States companies, this field contains your Taxpayer Identification Number. 
-This is a nine digit number that is usually called an EIN for an Employer Identification Number if this company is a corporation, 
-or SSN for a Social Security Number if this company is a person.
-This value is required if you subscribe to Avalara Managed Returns or the SST Certified Service Provider services, 
-but it is optional if you do not subscribe to either of those services.
+     * @var boolean TODO
      */
-    public $taxpayerIdNumber;
+    public $unusedMultiCert;
 
     /**
-     * @var boolean Set this flag to true to give this company its own unique tax profile.
-If this flag is true, this company will have its own Nexus, TaxRule, TaxCode, and Item definitions.
-If this flag is false, this company will inherit all profile values from its parent.
+     * @var float What is the exempt percentage from this certificate
      */
-    public $hasProfile;
+    public $exemptPercentage;
 
     /**
-     * @var boolean Set this flag to true if this company must file its own tax returns.
-For users who have Returns enabled, this flag turns on monthly Worksheet generation for the company.
+     * @var int TODO
      */
-    public $isReportingEntity;
+    public $verificationNumber;
 
     /**
-     * @var string If this company participates in Streamlined Sales Tax, this is the date when the company joined the SST program.
+     * @var int TODO
      */
-    public $sstEffectiveDate;
+    public $taxNumber;
 
     /**
-     * @var string The two character ISO-3166 country code of the default country for this company.
+     * @var boolean TODO
```

Comment:
We found quite a few in 17.7; they are being corrected in the 17.8 release, and we're adding an integration test that ensures that the word "TODO" never appears in the swagger file.